pkg/playground/plot-pf4.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import ReactDOM from "react-dom";
+import { ChartArea, ChartContainer, ChartGroup, ChartLabel, ChartVoronoiContainer } from '@patternfly/react-charts';
+
+const Sparkline = () => {
+    return (
+        <>
+            <h3>Vertical</h3>
+            <div className="sparky">
+                <div style={{ height: '100px', width: '400px' }}>
+                    <ChartGroup
+                        ariaDesc="Average number of pets"
+                        ariaTitle="Sparkline chart example"
+                        containerComponent={<ChartVoronoiContainer labels={({ datum }) => `${datum.name}: ${datum.y}`} constrainToVisibleArea />}
+                        height={100}
+                        maxDomain={{ y: 9 }}
+                        padding={0}
```

Comment:
Setting padding to 50, height to 200 (line 15) and width to 500 (line 18) make sure it doesn't distort the little tooltip popup.

Before:
![Screenshot from 2020-07-14 00-58-58](https://user-images.githubusercontent.com/209895/87361830-5f590c80-c56d-11ea-8ac3-1e922827c40b.png)

After:
![Screenshot from 2020-07-14 00-57-46](https://user-images.githubusercontent.com/209895/87361843-64b65700-c56d-11ea-8380-5e20de7b3f72.png)

Will look into the label positioning, because that's off suddenly.

Reply:
But we _can't_ do padding. (Or lines around the edges. Or probably lines at all.) That breaks things.

The label is _supposed to_ point to the area and be within the bounds. Hence the angled triangle.